dependencies.gradle

```diff
@@ -35,26 +35,41 @@ ext {
     ]
 
     //AndroidX versions
-    appCompatVersion = '1.3.1'
+    appCompatVersion = '1.4.1'
     constraintLayoutVersion = '2.1.1'
     ktxCoreVersion = '1.6.0'
     ktxFragmentVersion = '1.3.6'
     roomVersion = '2.3.0'
     webkitVersion = '1.4.0'
     windowManagerVersion = '1.0.0'
     lifecycleRuntimeKtxVersion = "2.4.0-rc01"
+    composeUiVersion = '1.1.1'
+    composeViewModelVersion = '2.4.1'
+    activityComposeVersion = "1.4.0"
+    constraintLayoutComposeVersion = "1.0.0-beta02"
```

Comment:
I would suggest maybe creating a new `composeDependencies` group so these don't mix with the normal views/androidx dependencies